android/src/main/java/com/vgscollectreactnative/VgsCollectReactNativeViewManager.kt

```diff
@@ -1,11 +1,13 @@
 package com.vgscollectreactnative
 
+import android.annotation.SuppressLint
```

Comment:
I would suggest against using suppresslint to remove lint warnings

Reply:
Done [here](https://github.com/Instawork/vgs-collect-react-native/pull/1/commits/65850574faec0cae00f0d2b793cf1529b5161f2f#diff-a41cc746377eae3513fca397551becca1eaaf1350440859511e60f1cb4dec243L3)